### DISCUSSION TITLE
Why is COMDIRAC separate from DIRAC

### DISCUSSION THREAD

**Discussion by marianne013** [DIRACGrid/DIRAC#6157] [Q&A] [answered]
We were asked to provide a functionality similar to COMDIRAC's "dls".
This has Simon and me wondering why the COMDIRAC commands do not form part of vanilla DIRAC as they seem to contain a number of useful functions. 
And given that we have successfully merged VMDIRAC and DIRAC, if there are any plans to merge COMDIRAC as well ?

**Comment by fstagni** [comment]
This is a topic for a developer meeting, but we can start discussion here:
- Apart from (better?) names for scripts, what are the functionalities that COMDIRAC has that are not already in DIRAC?
- What are the  duplicated functionalities? I am pretty sure there are some

Someone needs to act as COMDIRAC advocate.

**Comment by atsareg** [comment]
I think this is a good suggestion to merge COMDIRAC into vanilla DIRAC. This would be a good occasion to review several COMDIRAC implementations w.r.t. to FileCatalogCLI, for example. In COMDIRAC project there are also several suggestion for new commands. Note that COMDIRAC commands has differences with respect to vanilla DIRAC commands as they are using sessions for caching the configuration to make commands more reactive which is important in the interactive environment. But may be this feature should be rather extended to all the DIRAC commands. This needs a serious review.

**Comment by marianne013** [comment, marked as answer]
https://github.com/DIRACGrid/DIRAC/pull/6403

**Reply by marianne013** [reply]
End of year clean up: The pull request should do as an answer.

**Reply by marianne013** [reply]
Of course "General" as a category doesn't have "Answers". Will now try and change categories.
